linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig.pass.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
 # make existing entries pass
+# assumption: use a nominally small value for var_accounts_maximum_age_login_defs
+#             so as to not interfere.
 for acct in $(grep -ve ':\(!!\|\*\):' /etc/shadow | awk -F: '{print $1}'); do
-    chage -M 30 -d $(date +%Y-%m-%d) $acct
+    chage -M 1 -d $(date +%Y-%m-%d) $acct
```

Comment:
The check for `max_life_existing` actually checks that a user's maximum age is between min and max. So an arbitrarily small number won't work.